packages/common/src/schemas/json/lightdash-dbt-2.0.json

```diff
@@ -325,6 +343,13 @@
                                                             ]
                                                         }
                                                     ]
+                                                },
+                                                "groups": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "string",
+                                                        "minLength": 1
+                                                    }
```

Comment:
We can also set `maxItems` to 2 for the current specification.

```json
                                                "groups": {
                                                    "type": "array",
                                                    "items": {
                                                        "type": "string",
                                                        "minLength": 1
                                                    },
                                                    "maxItems": 2
                                                }
```

Reply:
could you add that pls?
And to each `groups`, this as well: 

```
  "description": "Groups are used to group dimensions and metrics in the sidebar. You can create nested groups up to 2 levels",
```